VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.html

```diff
@@ -28,34 +28,36 @@
 
 <div *ngIf="showVideo && isPrivateConsultation" class="video-background">
   <div class="header-bar">
-    <div class="govuk-grid-column-three-quarters govuk-body" style='float: left'>
-      <span class="room-title">
+    <div class="header-content-grid govuk-body">
+      <div class="room-title">
         {{getRoomName()}} 
-      </span>
-      <span class="room-status">
+      </div>
+      <div class="room-status">
         <span class="lock-badge" [class]="this.participant?.current_room?.locked ? 'locked' : 'unlocked'">{{this.participant?.current_room?.locked ? 'locked' : 'unlocked'}}</span>
-      </span>
-      <span *ngIf="!this.participant?.current_room?.locked" class="room-status-desc">
-          Other participants can enter the room. You can lock the room if you want to prevent anyone else entering.
-      </span>
-      <span *ngIf="this.participant?.current_room?.locked" class="room-status-desc">
-        Other participants cannot enter. You can invite others into the room by calling them <i class="fas fa-phone-alt" aria-hidden="true"></i>, or you can unlock the room to allow others to join.    </span>
-    </div>
-    <div class="govuk-grid-column-one-quarter">
-      <ng-container *ngIf="connected && showVideo">
-        <app-private-consultation-room-controls
-          [conferenceId]="conference.id"
-          [participant]="participant"
-          [audioOnly]="audioOnly"
-          [isPrivateConsultation]="isPrivateConsultation"
-          [outgoingStream]="outgoingStream"
-          [isSupportedBrowserForNetworkHealth]="isSupportedBrowserForNetworkHealth"
-          [showConsultationControls]="showConsultationControls"
-          (leaveConsultation)="showLeaveConsultationModal()"
-          (lockConsultation)="setRoomLock($event)">
-        </app-private-consultation-room-controls>
-      </ng-container>
+      </div>
+      <div *ngIf="!this.participant?.current_room?.locked" class="room-status-desc">
+        Lock the room to prevent others joining
+      </div>
+      <div *ngIf="this.participant?.current_room?.locked" class="room-status-desc">
+        Invite others by calling them or unlock the room for others to join
+      </div>
+      <div>
```

Comment:
I think we should remove this, I'll check it out over the weekend

Reply:
super thanks, ill merge monday. The grid is an improvement over the current and the fixes the ticket so im not too fussed.
In an ideal world, the buttons need to be changed but it would make this a quite bit bigger task.

Reply:
Removed extra div, still works correctly due to grid configuration